lambdas/app.js

```diff
@@ -38,8 +37,23 @@ const originRequest = async (event, context) => {
     });
 };
 
-const originResponse = async (event) => {
-  const { response } = event.Records[0].cf;
+const originResponse = async (event, context) => {
+  const { request, response } = event.Records[0].cf; 
+  if (['404', '403'].includes(response.status)) {
```

Comment:
why for 403s as well?

Reply:
@davemcorwin  - i was being consistent with the blog posts I have seen on SPA routing for lambda edge where this is performed for 404 and 403.  i can remove 403

Reply:
I'm not necessarily saying we need to get rid of it, i'm just wondering why. If it's included in the blog posts there must be a reason.

Reply:
> Why do we need to handle 403 as well? It is because this response code, instead of 404, is returned by Amazon S3 for assets that are not present. For instance, a url of https://yourdomain.com/somewhere will be looking for a file called somewhere (without extension) that does not exist.